bootloaders/riotboot/README.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+This folder contains a simple bootloader called "riotboot".
+A header with metadata, of length `RIOTBOOT_HDR_LEN` precedes
+the actual image. The header contains "RIOT" as a magic
+number to recognise a RIOT firmware image, a checksum, and
+the version of the RIOT firmware `APP_VER`.
+This bootloader verifies the checksum of the image which is located
+at an offset (`ROM_OFFSET`) with respect to  the `ROM_START_ADDR`
+defined by the CPU, just after the space allocated for riotboot.
```

Comment:
This documentation needs uptading. We are not verifying the checksum of the `image` put the metadata right ?
Also a bit of the same confusion between `image` and `slot` and `metadata`.

Reply:
Yes, I will try to read carefully everything and be consistent on the naming, since I agree it's confusing.